src/App.js

```diff
@@ -76,4 +72,21 @@ class App extends React.Component {
   }
 }
 
+class App extends React.Component {
+  render() {
+    return (
+      <>
+        <Router>
+          <Switch>
+          <Route exact path="/">
+            {LOGGEDIN ? <Redirect to="/experiment" /> : <Login/>}
+          </Route>
+            <Route path='/experiment' component={ExpStart} />
```

Comment:
Could we also make `ExpStart` a component in its own file? And maybe even move it together with `login.js` to a `components` folder?

Reply:
Technically, that has to do with the transpiler you use, but it is good practice